samples/ProjectGrandmaster.Unity/Assets/Free Low Poly Chess Set/Example/Scripts/CameraRotate.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraRotate : MonoBehaviour {
```

Comment:
nit: Curly brace on new line

Reply:
I should be able to remove some files from 'Free Low Poly Chess Set' as we're just using the chess piece prefabs.